server_installation/topics/operator/installation.adoc

```diff
@@ -0,0 +1,89 @@
+
+=== Installing {project_operator} into a cluster 
+
+To install the {project_operator} into 
+ifeval::[{project_community}==true]
+a Kubernetes cluster or
+endif::[]
+an Openshift cluster, you have two options:
+
+* The Operator Lifecycle Manager (OLM)
+* Command line installation
+
+==== Using Operator Lifecycle Manager
+
+For a new OpenShift 4.x cluster, the {project_operator} should already be available in
+OperatorHub tab of the OpenShift administration panel.
+
+image:{project_images}/operator-openshift-operatorhub.png[]
+
+ifeval::[{project_community}==true]
+For a Kubernetes cluster, perform these steps:
+
+. Go to link:https://operatorhub.io/operator/keycloak-operator[Keycloak Operator on OperatorHub.io].
+
+. Click `Install`.
+
+. Follow the instructions on the screen.
+
+image:{project_images}/operator-operatorhub-install.png[]
+
+endif::[]
+
+Once the {project_operator} is available in your cluster, click `Install` and follow the instruction on the screen.
+
+image:{project_images}/operator-olm-installation.png[]
+
+.Additional Resources
+
+ifeval::[{project_community}==true]
+For more information on a Kubernetes installation, see link:https://operatorhub.io/how-to-install-an-operator[How to install an Operator from OperatorHub.io].
+endif::[]
+
+For more information on an OpenShift installation, see the link:https://docs.openshift.com/container-platform/4.2/operators/olm-what-operators-are.html[OpenShift Operators guide].
+
+==== Installing from the command line
+
+You can install the {project_operator} at the command line using the link:{operatorRepo_link}[Github repo]:
+
+. Install all required Custom Resource Definitions by using the following command:
+
++
+```bash
+kubectl apply -f deploy/crds/
```

Comment:
This is an open question - you're proposing using `kubectl` for the commands. That's fine but the product should probably use `oc`.

Reply:
I think I need to use ifevals to show kubectl for the community.  However, if the community is using OpenShift, would oc be the valid command.

Reply:
@stianst @bdaw @hmlnarik WDYT? Maybe we should consistently use `oc` command and just put a comment somewhere that `oc create -f` is an equivalent of `kubectl apply -f`?

Reply:
I'd suggest variants being covered by macros defined in `topics/templates/document-attributes-product.adoc` and `topics/templates/document-attributes-community.adoc`, where community would use `kubectl` variant and product `oc` one.

Reply:
@hmlnarik What if a keycloak user is using the community version of OpenShift? Would it be oc instead?

Reply:
Actually, this topic says that kubectl works in most cases:   https://docs.openshift.com/container-platform/4.4/cli_reference/openshift_cli/usage-oc-kubectl.html
Do we know if that's true for the Operator? If so, we could stick with kubectl only.